tests/teams_integrations_test.py

```diff
@@ -62,6 +64,32 @@ def test_list_teams(mock_client, username, expected_team_names):
     unittest.TestCase().assertCountEqual(team_names_from_response, expected_team_names)
 
 
+def test_list_all_teams(mock_client):
+    response = mock_client.get(
+        "/teams?include=all",
+        headers=auth_header(get_bearer_token_for_user(kc_config.janedoe)),
+    )
+    assert response.status_code == 200
+    assert {team["name"] for team in response.json()} == set(kc_config.teams)
+
+
+def test_list_teams_is_member(mock_client):
+    response = mock_client.get(
+        "/teams?include=all",
+        headers=auth_header(get_bearer_token_for_user(kc_config.janedoe)),
+    )
+    assert response.status_code == 200
+
+    teams = {
+        name: list(team)[0]
+        for name, team in groupby(response.json(), itemgetter("name"))
+    }
```

Comment:
Sikkert en grunn, men ser ikke umiddelbart hvorfor dette ikke kan gjøres som `{team["name"]: team for team in response.json()}`? 😅